tests/integration/test_manylinux.py

```diff
@@ -406,7 +412,13 @@ def test_build_wheel_depending_on_library_with_rpath(any_manylinux_container, do
                 with w.open(name) as f:
                     elf = ELFFile(io.BytesIO(f.read()))
                     dynamic = elf.get_section_by_name('.dynamic')
+                    rpath = runpath = None
+                    for t in dynamic.iter_tags():
+                        if t.entry.d_tag == 'DT_RPATH':
+                            rpath = t
+                        elif t.entry.d_tag == 'DT_RUNPATH':
```

Comment:
Correct me if I'm wrong but I think the original logic could have left intact with the addition of an extra assertion:

`assert len([t for t in dynamic.iter_tags() if t.entry.d_tag == 'DT_RUNPATH']) == 0` instead of the ` assert runpath is None` line.

I think this is simpler and it also keeps the original assertion on having only one `DT_RPATH`, while the new change takes the last one in the list and doesn't check the length.